lfs/setup.go

```diff
@@ -28,11 +28,13 @@ var (
 		Properties: map[string]string{
 			"clean":    "git-lfs clean -- %f",
 			"smudge":   "git-lfs smudge -- %f",
+			"process":  "git-lfs filter",
```

Comment:
I wonder if we should call the `git-lfs` parameter `process` instead of `filter` to align the naming with Git?